docker/compose/mozdef_meteor/files/settings.js

```diff
@@ -12,7 +12,6 @@ mozdef = {
   port: "80",
```

Comment:
It's worth mentioning here that this file doesn't actually get used anymore in docker meteor environment (we write to the wrong location). This never resulted in errors because we have docker settings in the real settings.js file (for example, we point to the rest api container).

Reply:
I think we have a couple options:
1. Convert that file over to an environment variable file (similar to how it's done in cloudy mozdef) and then include that .env file in docker-compose.
2) Sync entire contents of https://github.com/mozilla/MozDef/blob/master/meteor/imports/settings.js with the file in the meteor container, however, that settings file now contains meteor specific logic such as https://github.com/mozilla/MozDef/blob/master/meteor/imports/settings.js#L31-L33, so it would be preferable to decouple https://github.com/mozilla/MozDef/blob/master/meteor/imports/settings.js#L10-L18 from the settings file in meteor, so we only overwrite the settings dict and nothing else. (We have a full version of that file with all of the meteor stuff in production, so we could use this functionality regardless)

Reply:
I think if we remove the file, then we're now saying "in order for docker to run, the meteor/imports/settings.js file must be using docker environment specific values" (which is what it currently is doing unless you use environment vars like cloudy mozdef) but I think it's preferable to keep the two configurations not dependent on each other.

Reply:
Probably rm the file and move to env variables is the right choice rather than having a one off file/source that we copy around.

Reply:
Agreed!

Reply:
OK, added some sanity checking for bool environment variables and cleaned up the way this all interfaces with the accounts system.